docs/fundamentals/how-do-blockchains-work.md

```diff
@@ -94,6 +94,10 @@ _Nodes_ are operated by ‘miners’ who are individuals or entities that have a
 
 ![distributed ledger technology](/img/how-do-blockchains-work/distributed-ledger-technology.jpeg)
 
+### The Nakamoto Consensus
```

Comment:
Do we intentionally have this section in 2 different places?

Reply:
Yes it's normal